### ISSUE TITLE
Is there any way to collect method chains?

### ISSUE THREAD

**Issue by callanchen** [rust-lang/rust-clippy#14700]
I'm writing a lint, but I want to get the names of all function calls in a chain.

How should I do this?

Such as:

```rust
let _ = S::build()
     .fun1()
     .fun2()
     .fun3();
```

I can get the Expr of "S::build()", but I can not find a good way to collect other function names in the chain.

**Comment by callanchen** [comment]
I find a help function in clippy_utils. I close the issue.
